Code/Mantid/instrument/MAPS_Parameters.xml

```diff
@@ -391,7 +391,7 @@
 </parameter>
 
 <!-- The semicolon separated list of possible log names, containing information on crystl rotation.
-     First found log will be used togethere with motor_offset to identify crystal rotation 
+     First found log will be used togethere with motor_offset to identify crystal rotation
```

Comment:
Typo here: togethere.
But considering the risks of changing/rebuilding, I'd leave it as it is, as I'd merge this in a while. It can be fixed in any follow-up ticket.